etna/datasets/tsdataset.py

```diff
@@ -279,13 +306,6 @@ def segments(self) -> List[str]:
         """
         return self.df.columns.get_level_values("segment").unique().tolist()
 
-    def _update_regressors(self):
-        result = set()
-        for column in self.columns.get_level_values("feature"):
-            if column.startswith("regressor"):
-                result.add(column)
-        self._regressors = list(result)
-
```

Comment:
What's going to it instead of the method?

Reply:
It is a matter of [this](https://github.com/tinkoff-ai/etna/issues/326) issue